lib/matplotlib/colors.py

```diff
@@ -643,6 +657,14 @@ def set_under(self, color='k', alpha=None):
         if self._isinit:
             self._set_extremes()
 
+    def get_over(self):
+        """
+        Get the color for high out-of-range values when ``norm.clip = False``.
```

Comment:
This still allows one to "get" the value, even without worrying about a `norm`. I'd suggest just dropping the "when ..." statement.

Reply:
I chose to mirror the docstring for `set_over` and it seems to me that both the getter and setter should match (either with or without that statement). The `norm` is a separate object from the color map, so a `norm` can't affect the return value of this method. I think the purpose of the "when..." statement is to clarify _when this value is used_ in context with a `norm` / figure.

I suggest removing the "when `norm.clip = False`" from both getter and setter if this is confusing. However, if you think it makes sense to keep the "when `norm.clip = False`" on the setter and remove it from the getter, I can do that -- just wanted to explain my reasoning before making that change.

Reply:
👍  I did not look at the docstring for the setters, so that makes sense!

I'd still have a slight preference for removing that from both of them since `norm` has its own docstring with the `clip` parameter described, but that is fine for another time too.

Reply:
I agree. I removed that phrase from all the get/set under/over docstrings in 8a36e2af3017833e695d80526f0fe3c9152e7212.